source/lib/formatter.ts

```diff
@@ -33,6 +38,19 @@ export default (diagnostics: Diagnostic[]): string => {
 
 		entry.errorCount++;
 		entry.messages.push(diagnostic);
+
+		if (showDiff && diagnostic.diff) {
+			let difference = diffStringsUnified(
+				diagnostic.diff.expected.replace(/'/g, '\'\''),
+				diagnostic.diff.received.replace(/'/g, '\'\''),
```

Comment:
Could the replacement go in `makeDiagnosticWithDiff()`?

Reply:
I removed the `UseSingleQuotesForStringLiteralType` flag in `typeToString` it's better because it keeps the default behavior of tsc so no need to type single quotes.